Danggn/Screen/Home/Model/ListDataModel.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ListDataModel.swift
+//  Danggn
+//
+//  Created by 김담인 on 2022/05/26.
+//
+
+import Foundation
+
+struct ListDataModel {
+    let listImage: String
+    let listTitle: String
+    let listLocation: String
+    let listPrice: String
+    let listLikeCount: Int
+}
+
+extension ListDataModel {
+    static let samleData: [ListDataModel] = [
```

Comment:
@daminoworld 
```suggestion
    static let sampleData: [ListDataModel] = [
```
오타발견!! ㅎㅎ